plugins/battery/battery.plugin.zsh

```diff
@@ -91,7 +91,7 @@ elif [[ $(uname) == "Linux"  ]] ; then
   }
 
   function battery_pct_prompt() {
-    b=$(battery_pct_remaining) 
+    b=$(battery_pct_remaining)
```

Comment:
This isn't a problem introduced by you, but it looks like both `$b` and `$color` in this function should be `local`. This may be a good opportunity to fix that.